src/app/services/question.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { environment } from '../../environments/environment';
+import { QuestionDto } from '../models/question-dto';
+import { QuestionQueueDto } from '../models/question-queue-dto';
+import { map } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class QuestionService {
+
+  constructor(private http: HttpClient) { }
+
+
+  getQuestionFromQueue(): Observable<QuestionQueueDto> {
```

Comment:
Rather than naming this based on the fact there's a queue, it would be better to name this method according to the intent, which is quarantine. Reason being that if we move away from a queue for some reason, you don't want to have to change this method name everywhere in the code base. Maybe something like `getQuestionFromQuarantine` ?

Reply:
I'll add an issue to renane all the queue references to quarantine